src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -3661,29 +3530,15 @@ public static void ConvertUnsignedToSigned()
         [Fact]
         public static void ConvertCheckedSignedToUnsigned()
         {
-            try
-            {
-                Func<ulong> f = Expression.Lambda<Func<ulong>>(Expression.ConvertChecked(Expression.Constant((sbyte)-1), typeof(ulong))).Compile();
-                ulong result = f();
-                Assert.False(true);
-            }
-            catch (OverflowException)
-            {
-            }
+            Func<ulong> f = Expression.Lambda<Func<ulong>>(Expression.ConvertChecked(Expression.Constant((sbyte)-1), typeof(ulong))).Compile();
+            Assert.Throws<OverflowException>(() => f());
         }
 
         [Fact]
         public static void ConvertCheckedUnsignedToSigned()
         {
-            try
-            {
-                Func<sbyte> f2 = Expression.Lambda<Func<sbyte>>(Expression.ConvertChecked(Expression.Constant(UInt64.MaxValue), typeof(sbyte))).Compile();
-                sbyte result = f2();
-                Assert.False(true);
-            }
-            catch (OverflowException)
-            {
-            }
+            Func<sbyte> f2 = Expression.Lambda<Func<sbyte>>(Expression.ConvertChecked(Expression.Constant(UInt64.MaxValue), typeof(sbyte))).Compile();
```

Comment:
That there is (and when one goes to touch the code it turns out that while they hit massive ranges of values, the coverage could probably still be better). I'm guessing it's a matter of copy-and-paste from somewhere that had both an `f` and an `f2`. In any case I've changed this and that at line 3526 and squashed it into the single commit.